app/models/site_status.rb

```diff
@@ -34,4 +34,20 @@ class SiteStatus < ApplicationRecord
   def recruitment_cycle
     "2019"
   end
+
+  def findable?
+    status == "Running" && publish == 'Y'
```

Comment:
If we define an enum for "published", I'm pretty sure this can be achieved in SQL with a scope like this:

```
scope :findable, -> { Running.published }
```

(although, I say we rename `Running` to `running` ...)

And similarly, we should be able to create the same method as you have:

```
def findable?
  Running? && published?
end
```

Reply:
We can do these later with the tickets:

1. rename enums to not be capitalised and without spaces
2. add enum for `publish` column
3. refactor `findable?` in `Course` and `SiteStatus` model to use the scopes generated by the enums

Reply:
Carded

https://trello.com/c/iUyp5KXW/995-ruby-enums-refactor\